techStack.md

```diff
@@ -9,20 +10,26 @@
 - [Firebase](https://firebase.google.com/) as a database for MVP projects
 - [Puppeteer](https://developers.google.com/web/tools/puppeteer/) for web-scraping
 
-## Automatic deployment
+## Experience
 
-- [Circle CI](https://circleci.com/) automatically deploys the code once a build is successful
-- We deploy websites using [Firebase](https://firebase.google.com/) website deployments
-- Server code / [Docker](https://www.docker.com/) containers are deployed using [Now](https://zeit.co/now) and [Heroku](https://www.heroku.com/)
+- We've put 20+ React / React Native / Redux / Node applications into production
+- We've built [tools](https://github.com/z-dev/rn-scaffold) to start best practice React Native codebases quickly
+- We have common components we reuse between projects - for example our [credit card checkout component](https://github.com/z-dev/react-native-checkout)
 
 ## Code Quality
-- We code review using Pull Requests on [Github](https://github.com/) / [Bitbucket](https://bitbucket.org/) / [Gitlab](https://gitlab.com)
-- We [lint](https://eslint.org/) all our code to check for common mistakes
+- Senior developers review all our code using Pull Requests on [Github](https://github.com/) / [Bitbucket](https://bitbucket.org/) / [Gitlab](https://gitlab.com)
```

Comment:
It's super minor, but this sentence might give the impression that someone outside the company reviews our code.

I would maybe say:

All code is reviewed by our senior developers via/using (<= delete as appropriate) Pull Requests on [Github](https://github.com/) / [Bitbucket](https://bitbucket.org/) / [Gitlab](https://gitlab.com)